src/database.go

```diff
@@ -0,0 +1,51 @@
+package src
```

Comment:
```suggestion
package database
```

sposterei quindi in una cartellina /database

Poi ti aggiungo sotto altro commento